joomla/joomla-cms
# [4.0] Namespace System - Privacy Consent plugin

### Summary of Changes

Namespaces System - Privacy Consent plugin.

### Testing Instructions

Apply patch. Delete `libraries/autoload_psr4.php` file.
Enable System - Confirm Consent plugin.
Login to frontend.
When redirected to profile form to accept consent check that Privacy Policy field appears correctly.

### Documentation Changes Required

IDK.
